README.md

```diff
@@ -1,3 +1,27 @@
-# caliber-staging-module-service
-Spring Boot Microservice for Caliber Staging Module
+# Spring Boot Microservice for Caliber Staging Module
 
+## Revature Project #3
+
+### Team Members:
+
+### Summary
+
+#### User Stories
+- A manager can view all associates assigned to him
+- A manager can view all associates that are new to staging
+- The system will automatically assign associates to a staging manager
+- A manager can create a SWOT analysis for an associate in order to decide actions the associate should prioritize
+
+**Architecture Diagram**
+![](./imgs/Staging-Module-AD.png)
+
+![](./imgs/Staging-Module-ERD.PNG)
+**ERD Diagram**
```

Comment:
@mareo1997 in the directory 'imgs' you can add the png files.  Then the README doc will know what image you're referring to

Reply:
You can add this folder directly into the project -- the same location where the README.md file is.  THEN, put the png images inside this folder,  push your changes, then I will re-review this merge request

Reply:
I thought I already did that. Like you did in project 1.

Reply:
For erd diagram if i try to put ![](./imgs/Staging-Module-ERD.PNG) under the label, the format doesn't match. So I just left it there